### PR TITLE
Tutorial: Integrating WalletConnect into your DApp

### LINKED ISSUE #6909: [WalletConnect] documentation for dApp integration
- Currently there's a tutorial for creating governance app (useContractKit), more tutorials needed.
- Can also go all the way down to Walletconnect level re: QR codes

### PR DESCRIPTION
**Pull request by AlexBHarley**
### Description

Not much to say but a quick tutorial on connecting DApps to WalletConnect compatible wallets without the use of use-contractkit.

### Other changes

Also bumped up the version of WC to `beta.0` from `alpha.38`.

### Related issues

Closes #6909

### CODE REVIEW

**Comment by critesjosh** [comment]
wallet connect is not yet live, correct? we should just sit on this until it launches?

**Comment by AlexBHarley** [comment]
@critesjosh yeah production relay servers should be up today. Once they're up, this document can go live as it's not necessarily tied to Valora. Support should land in Valora next release.

**Comment by critesjosh** [comment]
Great! Let me know when they're up and I'll go through the guide

File: packages/docs/developer-resources/walkthroughs/wallet-connect.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  // send transactions
+  const accounts = await wallet.getAccounts()
+}
+
+connect()
+```
+
+## Initialise ContractKit
+
+As a final step, `WalletConnectWallet` (and any library that implements the `Wallet` interface we mentioned earlier) integrates natively with [@elo/contractkit](https://docs.celo.org/developer-guide/contractkit). This means you can start sending various transactions as soon as the wallet is initialised. Here's a full example displaying the end-to-end flow of connecting and sending transferring some CELO.
```
Comment (critesjosh) [inline packages/docs/developer-resources/walkthroughs/wallet-connect.md]:
```suggestion
As a final step, `WalletConnectWallet` (and any library that implements the `Wallet` interface we mentioned earlier) integrates natively with [@celo/contractkit](https://docs.celo.org/developer-guide/contractkit). This means you can start sending various transactions as soon as the wallet is initialised. Here's a full example displaying the end-to-end flow of connecting and sending transferring some CELO.
```

**Comment by critesjosh** [comment]
@AlexBHarley I just tried going through this with no luck. Are the relay servers not up yet?

**Review by critesjosh — APPROVED** [review]
Tested as a node.js script, I was able to get the URI from wallet connect. I wasn't able to test the connection to Valora or the test wallet since it's not implemented there yet.

**Pull request merged**